docs/endpoints.rst

```diff
@@ -1021,9 +1021,225 @@ Show number of users editing buildings before, during and after Nepal earthquake
       }
 
 
-Extraction Endpoints
---------------------
+Contributions Aggregation
+-------------------------
+      
+.. http:post :: /contributions/count
+
+   Get the count of the contributions provided to the OSM data. This endpoint does not support the deprecated ``types``, ``keys``, ``values`` parameters.
+
+**Example request**:
+
+Number of contributions to the building 'Stadthalle Heidelberg' between 2010 and 2020.
+
+  .. tabs::
+
+    .. code-tab:: bash curl (GET)
+
+       curl -X GET 'https://api.ohsome.org/v1/contributions/count?bboxes=8.699053,49.411842,8.701311,49.412893&filter=id:way/140112810&time=2010-01-01,2020-01-01
```

Comment:
The forgotten single quote at the end of this URL causes a warning at line 1037 creating the docs with Sphinx

Reply:
Good catch.. my sphinx did not give that specific warning.. just something cryptic about line 4 in this file, but after that fix the warning went away 🤷